src/converters/betterdiscord.ts

```diff
@@ -8,6 +9,10 @@ class BDWebpackApi implements IBaseWebpackApi {
     }
 }
 
+export function convertFormat(ast: Statement[]) {
```

Comment:
If this not being used for anything. Comment it out or remove it.

Reply:
https://github.com/LosersUnited/DiscordModCompiler/pull/10/files#diff-2c718087c1fd72979602bd9da34119fb956d070d90109b6bc01ea23b2ad7eae1

Reply:
What is this even being used for?

Reply:
after committing the change I noticed its not needed at the moment, perhaps I will populate this later, in another pr